### ISSUE TITLE
Custom mutators don't work anymore on 4.5.2 when setting data

### ISSUE THREAD

**Issue by yuukuari** [olifolkerd/tabulator#2520]
Hi,

I upgraded Tabulator from 4.4.1 to 4.5.2 and custom mutators don't work anymore.

With following simple example, the mutator won't be called and the `mutated` field will always be empty.
```
const mutatorTable = new Tabulator('#mutatorTable', {
    height: '100%',
    columns: [
        {title: 'Not mutated', field: 'notmutated'},
        {title: 'Mutated', field: 'mutated', mutator: () => 'mutated'}
    ]
})

mutatorTable.setData([
    { notmutated: 1 }
])
```

PS : I have checked that the mutator module is installed and it is.

New features from 4.5 are great.
Thanks !

**Comment by yuukuari** [comment]
Might come from that commit :
https://github.com/olifolkerd/tabulator/commit/84c15c8a7a0d825fae0993598cbcbec706876dbe

**Comment by olifolkerd** [comment]
Hey @yuukuari 

If you are reporting a bug, please follow the [Bug Reporting Guide](http://tabulator.info/community#bug) and use the bug issue template as it prompts you for the information we need to help you

you ***MUST*** provide a link to a JS Fiddle or Code Pen that demonstrates your issue with a step by step by step guide to replicate the issue.

Cheers

Oli :)

**Comment by yuukuari** [comment]
Sorry,

Here the fiddle links :
- mutator with Tabulator 4.4.3 : https://jsfiddle.net/hzvdp4tu/
- mutator with Tabulator 4.5.2 : https://jsfiddle.net/hzvdp4tu/1/

The steps to reproduce the issue are as follow :
- The table is created with 3 columns : HT price, VAT and TTC price which is computed using the HT price and VAT
- Data is added to the table using `setData` **without specifying any TTC value**

With Tabulator 4.4.3, the TTC value is computed using the defined mutator
With Tabulator 4.5.2, the TTC is never computed because the mutator is not called.

With this new version, the only way to call the mutator is as you describe, to explicitly set a TTC value to each rows. But it was not the case before, and it makes it difficult for me to upgrade to that new version because i have many, many tables with mutators set this way.

Hope that helps,
Yuuk

**Comment by britvik** [comment]
I noticed the same behavior. Is this by design now? What is the point of mutators then? If I have to modify the data before I post them to tabulator I see no point in using mutators.

**Comment by olifolkerd** [comment]
Hey @yuukuari 

Thanks for that, yes that is a bug, they should be called on the column when the data is set but they wont be called on the column again after an edit unless the value in that column is explictly changed to prevent accidental double mutation

Cheers

Oli :)

**Comment by olifolkerd** [comment]
i have pushed a fix for this to the master branch, it will be included in the 5.4.3 release

**Comment by olifolkerd** [comment]
4.5.3 is released now :)
